.github/labeler.yml

```diff
@@ -0,0 +1,12 @@
+# if a label exists in nixpkgs then use that instead of making a new one
+
+"8.has: changelog":
+  - /doc/manual/src/release-notes/*
+
```

Comment:
```suggestion
```

Since we currently don't have a dedicated label and try to keep their number down, keep it out for now.